WebRoot/js/zimbraMail/prefs/view/ZmPriorityMessageFilterDialog.js

```diff
@@ -68,6 +68,7 @@ function() {
 	
 	this._moveMsgIntoStream = this._priorityMessageForm.getControl("MOVE_MSG_STREAM");
 	this._notToMe = this._priorityMessageForm.getControl("NOT_TO_ME");
+	this._notToMeCont = document.getElementById(params.id + "_NOT_TO_ME_CONT");
```

Comment:
Dwt.byId can be used, instead of making reference to 'document' global variable.